python/example/brunel.py

```diff
@@ -0,0 +1,161 @@
+            connections.append(arbor.connection((i,0), (gid,0), self.weight_inh_, self.delay_))
+        return connections
+
+    def cell_description(self, gid):
+        cell = arbor.lif_cell()
+        cell.tau_m = 10
+        cell.V_th = 10
+        cell.C_m = 20
+        cell.E_L = 0
+        cell.V_m = 0
+        cell.V_reset = 0
+        cell.t_ref = 2
+        return cell
+
+    def event_generators(self, gid):
+        t0 = 0
+        sched = arbor.poisson_schedule(t0, self.lambda_, gid + self.seed_)
+        return [arbor.event_generator(arbor.cell_member(gid,0), self.weight_ext_, sched)]
+
+    def num_targets(self, gid):
+        return 1
+
+    def num_sources(self, gid):
+        return 1
+
+class options:
+    pass
+
+if __name__ == "__main__":
+    opt=options()
+
+    parser = argparse.ArgumentParser(description='Brunel model miniapp.')
+    parser.add_argument('-n', '--n-excitatory', dest='nexc', type=int, default=400, help='Number of cells in the excitatory population')
+    parser.add_argument('-m', '--n-inhibitory', dest='ninh', type=int, default=100, help='Number of cells in the inhibitory population')
+    parser.add_argument('-e', '--n-external', dest='next', type=int, default=40, help='Number of incoming Poisson (external) connections per cell')
+    parser.add_argument('-p', '--in-degree-prop', dest='syn_per_cell_prop', type=float, default=0.05, help='Proportion of the connections received per cell')
+    parser.add_argument('-w', '--weight', dest='weight', type=float, default=1.2, help='Weight of excitatory connections')
+    parser.add_argument('-d', '--delay', dest='delay', type=float, default=0.1, help='Delay of all connections')
+    parser.add_argument('-g', '--rel-inh-w', dest='rel_inh_strength', type=float, default=1, help='Relative strength of inhibitory synapses with respect to the excitatory ones')
+    parser.add_argument('-l', '--lambda', dest='poiss_lambda', type=float, default=1, help='Expected number of spikes from a single poisson cell (kHz)')
+    parser.add_argument('-t', '--tfinal', dest='tfinal', type=float, default=100, help='Length of the simulation period (ms)')
+    parser.add_argument('-s', '--dt', dest='dt', type=float, default=1, help='Simulation time step (ms)')
+    parser.add_argument('-G', '--group-size', dest='group_size', type=int, default=10, help='Number of cells per cell group')
+    parser.add_argument('-S', '--seed', dest='seed', type=int, default=42, help='Seed for poisson spike generators')
+    parser.add_argument('-f', '--write-spikes', dest='spike_file_output', action='store_true', help='Save spikes to file')
+    parser.add_argument('-z', '--profile-rank-zero', dest='profile_only_zero', action='store_true', help='Only output profile information for rank 0')
+    parser.add_argument('-v', '--verbose', dest='verbose', action='store_true', help='Print more verbose information to stdout')
```

Comment:
These two options `--profile-rank-zero` and `--verbose` don't actually do anything in this example.

Reply:
Correct. This is inherited from `brunel.cpp` ;)